src/sentry/utils/snuba.py

```diff
@@ -346,13 +346,16 @@ def get_project_issues(project_ids, issue_ids=None):
             tombstone.project_id, {}
         )[tombstone.hash] = tombstone.deleted_at
 
-    # return [(gid, [(hash, tombstone_date), (hash, tombstone_date), ...]), ...]
+    # return [(gid, pid, [(hash, tombstone_date), (hash, tombstone_date), ...]), ...]
     result = {}
     for h in hashes:
         tombstone_date = tombstones_by_project.get(h.project_id, {}).get(h.hash, None)
-        pair = (h.hash, tombstone_date.strftime("%Y-%m-%d %H:%M:%S") if tombstone_date else None)
-        result.setdefault(h.group_id, []).append(pair)
-    return list(result.items())[:MAX_ISSUES]
+        issue = (
```

Comment:
minor naming nit, I still think this is maybe best called `pair` or something other than `issue` as it's not really an issue, but just a single hash/tombstone tuple.